client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java

```diff
@@ -93,19 +92,28 @@
     @UiField(provided = true)
     BrandPicker brandPicker;
     @UiField
-    LIElement smallAnchor;
+    LIElement smallLI;
```

Comment:
Not sure that I like the LI in the name. If you want to add something to small, why not smallChoice or smallSize ?